.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: node_js
```

Comment:
Do you know if its possible to make Travis CI run on individual packages in our monorepo?

Reply:
(Sorry for the irrelevant comment, I thought this package was actually _in_ our monorepo. Still wondering if you know its possible though 😇)

Reply:
Sort of. I was thinking we might bootstrap and then specify explicitly which packages to run tests on as `cd packages/@sanity/<some-pkg> && npm test`. If you catch my drift.